src/main/java/org/apache/commons/text/StrTokenizer.java

```diff
@@ -842,10 +838,9 @@ private int readWithQuotes(final char[] srcChars, final int start, final int len
                     continue;
                 }
 
-                // copy regular character from outside quotes
-                workArea.append(srcChars[pos++]);
-                trimStart = workArea.size();
             }
+            workArea.append(srcChars[pos++]);
```

Comment:
@arturobernalg can you add back the comment above these lines, please?

Reply:
changed